man/fi_rxm.7.md

```diff
@@ -88,6 +88,11 @@ RxM provider does not support the following features:
 RxM provider should work fine for client - server programs like fabtests. Support for MPI, SHMEM
 and other applications is work in progress.
 
+## Known issues
+
+Applications that use fi_cq_sread or wait on CQ fd may not function correctly. This would be fixed
+in the next release.
```

Comment:
I could return -FI_ENOSYS for the cq_sread/sreadfrom calls until they are fixed.